charts/portworx/values.yaml

```diff
@@ -50,13 +50,17 @@ miscArgs: none                        # Miscellaneous arguments that will be pas
 
 disableStorageClass: false            # Instructs Operator to not install the default Portworx StorageClasses.
 
-stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
-storkVersion:                         # Optional: version of Stork. For eg: 2.7.0, when it's empty Portworx operator will pick up
+stork:                                # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
+  enabled: true
+  storkVersion:                       # version of Stork. For eg: 2.7.0, when it's empty Portworx operator will pick up
```

Comment:
what's the reason that we are not taking input of image of stork and rather only version? what will happen in case of any custom images? CC : @diptiranjanpx

Reply:
We should accept registry, imagename and imagetag(version). 
Imagename will mostly be `stork` always but the other two should be configurable.

Reply:
@nikita-bhatia  @diptiranjanpx registry are configurable through field `customImageRegistryURL`. [This](https://github.com/portworx/helm/blob/master/charts/portworx/templates/_helpers.tpl#L71) helper func will format complete image with version that we passed in `stokVersion`